### ISSUE TITLE
{Linked:Bug4815270}[Adaptive Cards > Designer] - In dark mode, the color contrast ratio of 'Learn more', w.r.t background color is 2.23:1, which is less than 4.5:1

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9070]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
Version 24H2(OS build 26100.2033)
Edge dev Version 132.0.2945.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'
Enable the Dark Mode.

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Open the tool and check the contrast ratio for the Learn more link and observe the issue.

**Actual Result:**
In dark mode, the color contrast ratio of 'Learn more', w.r.t background color is 2.23:1, which is less than 4.5:1.

**Expected Result:**
In dark mode, the contrast ratio of Learn more link should be greater than or equal to 4.5:1._

**User Impact:**
Most people find it easier to read text when it has a sufficiently high contrast against its background. People with visual disabilities, low vision, limited color perception, or [presbyopia](https://en.wikipedia.org/wiki/Presbyopia) are likely to find text unreadable when contrast is too low.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/3a90dd5c-e01d-41dc-94bb-689987d205ba)

[json file for bug.txt](https://github.com/user-attachments/files/18080630/json.file.for.bug.txt)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler 

Could you please provide us latest update on this bug as this bug is OSLA.

**Comment by anna-dingler** [comment]
Hi @vagpt,

@satwiksharma is taking over accessibility work.

Thanks!
Anna

**Comment by SatwikKrSharma** [comment]
@vagpt this issue is resolved

![Image](https://github.com/user-attachments/assets/41f6275b-629a-4201-9a09-778bde91fdf8)

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

![Image](https://github.com/user-attachments/assets/3e327a24-66b0-49fd-b667-bc3a3151d407)
